server/manager/src/main/java/org/apache/accumulo/manager/tableOps/split/FindSplits.java

```diff
@@ -66,22 +91,68 @@ public Repo<Manager> call(FateId fateId, Manager manager) throws Exception {
           tabletMetadata.getLogs().size());
```

Comment:
This is preexisting problem, seems like it should return null here.

```suggestion
          tabletMetadata.getLogs().size());
          return null;
```